docs/bigquery-usage.rst

```diff
@@ -34,31 +34,74 @@ Authorization / Configuration
      >>> client = bigquery.Client()
 
 - Override the credentials inferred from the environment by passing explicit
-  ``credentials`` to one of the alternative classmethods, `from_service_account_json`:
+  ``credentials`` to one of the alternative ``classmethod`` factories,
+  `:meth:gcloud.bigquery.client.Client.from_service_account_json`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_json('/path/to/creds.json')
 
-  or `from_service_account_p12`:
+  or `:meth:gcloud.bigquery.client.Client.from_service_account_p12`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_p12('/path/to/creds.p12', 'jrandom@example.com')
 
-- Override the project inferred from the environment by passing an explicit
-  ``project`` to the constructor, or to either of the alternative classmethods:
+
+Projects
```

Comment:
I think we should map the API surface. If putting `list_projects` on a `Client` is confusing, we should put it somewhere else.

@jgeewax ISTM that this would be different than the Resource Manager API. Like it would give you access to any projects which you had an ACL on (but not necessarily access to the project in the Cloud Console).

From the [discovery doc][1], it would seem it is no different than resource manager:

> bigquery.projects.list ... Lists all projects to which you have been granted any project role.

I'm not sure what to think here (about this functionality vs. Resource Manager).

[1]: https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest